assets/xml/scenes/overworld/spot15.xml

```diff
@@ -1,5 +1,7 @@
 <Root>
     <File Name="spot15_scene" Segment="2">
+        <Cutscene Name="gHyruleCastleIntroCs" Offset="0x3F40"/>
+        <!--<Cutscene Name="gHyruleCastleIntro2Cs" Offset="0x4280"/> There is an entry in the Entrance table but its not a valid cutscene -->
```

Comment:
What is this referring to? I don't see any corresponding entry in the table but I might be looking at the wrong thing